packages/core/crates/core-mixer/src/mixer.rs

```diff
@@ -1,38 +1,38 @@
 use std::time::Duration;
 
-use futures::stream::{Stream};
+use futures::stream::Stream;
 use futures_lite::stream::StreamExt;
 use rand::Rng;
 
 use crate::future_extensions::StreamThenConcurrentExt;
 
-
 #[cfg(not(wasm))]
-use async_std::task::sleep as sleep;
+use async_std::task::sleep;
 
 #[cfg(wasm)]
-use gloo_timers::future::sleep as sleep;
-
-
-const DELAY_MIN_MS: u64 = 0;    /// minimum delay in milliseconds
-const DELAY_MAX_MS: u64 = 200;  /// maximum delay in milliseconds
+use gloo_timers::future::sleep;
 
+const DELAY_MIN_MS: u64 = 0;
+/// minimum delay in milliseconds
```

Comment:
Are you sure these comments are correct? They should be above their reference.

Reply:
that's indeed weird ...